inspector/src/pages/post-debugger.ts

```diff
@@ -145,7 +147,14 @@ function createImportFileButton(
         } else {
           indexOfEnd = indexOfBrk + offset;
         }
-        const logLine = remaininStrConsidered.substring(0, indexOfEnd);
+        let logLine = remaininStrConsidered.substring(0, indexOfEnd);
+  
+        if(isInitLog(logLine)) {
+          let init = parseAndGenerateInitLog(logLine)
+          logLine = init.log
+          timeAtPageLoad = init.timeAtPageLoad
+          dateAtPageLoad = init.dateAtPageLoad
```

Comment:
dateAtPageLoad = init.dateAtInit - init.timeAtInit

Reply:
I added it in the "parseAndGenerateInitLog" fn